kit/ForKit.cpp

```diff
@@ -470,12 +470,8 @@ void forkLibreOfficeKit(const std::string& childRoot,
     // Cleanup first, to reduce disk load.
     cleanupChildren();
 
-#ifndef KIT_IN_PROCESS
-    (void) limit;
-#else
-    if (limit > 0)
+    if (!Util::isKitInProcess() && limit > 0)
```

Comment:
The condition equivalent to the original code should be "if (Util::isKitInProcess() && limit > 0)".

Reply:
ah good catch =) thanks will fix.